tests/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+find_package(GTest REQUIRED)
```

Comment:
GTest should be included via EDM/CPM. Have a look at https://github.com/EVerest/libevse-security for example

Reply:
@andistorm This library doesn't currently use EDM/CPM. I'm concerned about adding the overhead to this project just to add a test dependency.

Reply:
agreed :+1: